### PR TITLE
Included description in the info that gets printed for models

### PR DESCRIPTION
**Pull request by KevinCLydon**
Now all those nice descriptions from cellarium-ai/cellarium-cloud#166 will be visible to you, the user

### CODE REVIEW

**Review by fedorgrab — APPROVED** [review]

File: cellarium/cas/settings/development.py [hunk 1]
```diff
@@ -1 +1,2 @@
-CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api-dev.cellarium.ai"
+# CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api-dev.cellarium.ai"
+CELLARIUM_CLOUD_BACKEND_URL = "http://localhost:8000"
```
Comment (nmalfroy) [inline cellarium/cas/settings/development.py:2]: Did you mean to leave this?

**Review by nmalfroy — CHANGES_REQUESTED** [review]
Happy to switch to an approve but I _think_ the change to the dev settings was by accident?

File: cellarium/cas/settings/development.py (hunk 1 above)
Comment (KevinCLydon) [reply cellarium/cas/settings/development.py:2]: Oh yup, you're right.  I keep forgetting to switch that back.  I'll fix it

**Review by nmalfroy — APPROVED** [review]
Looks good!

**Pull request merged**